### ISSUE TITLE
Showroom crashes

### ISSUE THREAD

**Issue by ViliamVadocz** [RLBot/RLBot#545]
Steps to reproduce:
- Launch RLBotGUI
- Edit appearance of any bot
- Click View car in game

https://discord.com/channels/348658686962696195/348659150793736193/809434951190577184

Log:
```
           ______ _     ______       _
     10100 | ___ \ |    | ___ \     | |   00101
    110011 | |_/ / |    | |_/ / ___ | |_  110011
  00110110 |    /| |    | ___ \/ _ \| __| 01101100
    010010 | |\ \| |____| |_/ / (_) | |_  010010
     10010 \_| \_\_____/\____/ \___/ \__| 01001



Version 1.55.1

    Using sockets for match start, lifting the restrictions on max players when using flatbuffers,
    and sending a game tick packet immediately upon socket connection to fix some bugs.

Trouble? Ask on Discord at https://discord.gg/5cNbXgG or report an issue at https://github.com/RLBot/RLBot/issues

Launching RLBot binary with args ['C:\\Users\\vilia\\AppData\\Local\\RLBotGUIX\\venv\\lib\\site-packages\\rlbot\\dll\\RLBot.exe', '23233', '127.0.0.1', '0']
2021-02-11 15:47:20,706 INFO:rlbot[    setup_manager.py:361 - ensure_rlbot_gateway_started() ] Python started RLBot.exe with process id 15708 and port 23233
Setting up soccer geometry for ball prediction...
Ball Prediction Service Started
RLBot.exe trying to connect to Rocket League on port 23233...
2021-02-11 15:47:21,123 INFO:rlbot[    setup_manager.py:238 - launch_rocket_league() ] Launching Rocket League with: ['c:\\program files (x86)\\steam\\steam.exe', '-applaunch', '252950', '-rlbot', 'RLBot_ControllerURL=127.0.0.1:23233', 'RLBot_PacketSendRate=240', '-nomovie']
2021-02-11 15:47:21,126 INFO:rlbot[    setup_manager.py:514 -          start_match() ] Python attempting to start match.RLBot.exe Connected successfully to port 23233 !
front end listening for connections
RLBot.exe has begun listening for local socket clients on port 23234.
front end server accepted new bot connection
ready to accept the next connection...
2021-02-11 15:47:43,506 INFO:socket_man[   socket_manager.py:120 -      connect_and_run() ] Connected!
Handling match start message which originated locally and which will apply locally.
Starting match with command open Stadium_P?game=TAGame.GameInfo_Soccar_TA?noreplay?GameTags=PlayerCount8,UnlimitedTime,UnlimitedBooster
Spawning bot 0 on team 0 with primary color rgb(6, 22, 127). Is rlbot? 1
uninitialized_bot error reading header: 2021-02-11 15:47:44,020 INFO:  vpd[valid_packet_detector.py:22 - wait_until_valid_packet() ] Waiting for valid packet...
An existing connection was forcibly closed by the remote host2021-02-11 15:47:44,021 INFO:socket_man[   socket_manager.py:120 -      connect_and_run() ] Connected!

front end server accepted new bot connection
ready to accept the next connection...
Game spawn detected at payload count 1
Zeroing numCars because we just got a fresh game spawn.
Clearing actor_id mappings!
Setting up soccer geometry for ball prediction...
Field spawn messages at payload count 1
We have a player spawn message ready to go, sending!
We had player spawn messages queued up when arriving at a pause menu, sent!
Detected player spawn. Current network role: 0
Got player spawn message for Showroom, actor id 38, command id 2
Associating player id 38 with player index 0!
2021-02-11 15:47:46,441 INFO:  vpd[valid_packet_detector.py:39 - wait_until_valid_packet() ] Packets are looking good, all spawn ids accounted for!
2021-02-11 15:47:46,442 INFO:rlbot[    setup_manager.py:517 -          start_match() ] Match has started
Traceback (most recent call last):
  File "src\\gevent\\greenlet.py", line 854, in gevent._gevent_cgreenlet.Greenlet.run
  File "C:\Users\vilia\AppData\Local\RLBotGUIX\venv\lib\site-packages\eel\__init__.py", line 265, in _process_message
    return_val = _exposed_functions[message['name']](*message['args'])
  File "C:\Users\vilia\AppData\Local\RLBotGUIX\venv\lib\site-packages\rlbot_gui\gui.py", line 298, in spawn_car_for_viewing
    spawn_car_in_showroom(loadout_config, team, showcase_type, map_name, launcher_prefs)
  File "C:\Users\vilia\AppData\Local\RLBotGUIX\venv\lib\site-packages\rlbot_gui\match_runner\match_runner.py", line 110, in spawn_car_in_showroom
    sm.game_interface.update_player_input(player_input, 0)
  File "C:\Users\vilia\AppData\Local\RLBotGUIX\venv\lib\site-packages\rlbot\utils\structures\game_interface.py", line 207, in update_player_input
    rlbot_status = self.game.UpdatePlayerInput(player_input, index)
AttributeError: 'NoneType' object has no attribute 'UpdatePlayerInput'
2021-02-11T14:47:46Z <Greenlet at 0x2797c552ae8: _process_message({'call': 12.683196862630588, 'name': 'spawn_car_fo, <geventwebsocket.websocket.WebSocket object at 0x0)> failed with AttributeError

uninitialized_bot error reading header: An existing connection was forcibly closed by the remote host
```

**Comment by ViliamVadocz** [comment]
Same error as in #543 after the fix: 
```
  File "...\rlbot\utils\structures\game_interface.py", line 207, in update_player_input
    rlbot_status = self.game.UpdatePlayerInput(player_input, index)
AttributeError: 'NoneType' object has no attribute 'UpdatePlayerInput'
```

**Comment by tarehart** [comment]
Thanks for reporting! Fixed with 1.55.2
